release.go

```diff
@@ -82,6 +95,18 @@ func (r Release) Version() string {
 	return r.version
 }
 
+func aggregateReleaseTimestamp(bundles []Bundle) (string, error) {
+	var t time.Time
+
+	for _, b := range bundles {
+		if b.Time.After(t) {
+			t = b.Time
+		}
+	}
+
+	return t.Format("2006-01-02T15:04:05.000000Z"), nil
```

Comment:
The format is defined by the spec. See https://github.com/giantswarm/api-spec/pull/54.